README.md

```diff
@@ -1,40 +1,41 @@
-planck
+# planck
+
+planck is an open-source email client for Android with encryption.
+
+## How to start development
+### Pre-requisistes
+OS: MacOS or Linux.
+The following dependencies are needed to build the project:
+- openjdk@11
```

Comment:
I see this line requires Java 11, but most conf files in the project are Java 8, also the sample. Which is correct?

Reply:
Could you please mention some files that are Java 8?

Reply:
I see some travis.yml but that is some old file.

Reply:
Ah I see there is some jdk8 in a sample local.conf

Reply:
updated.

Reply:
exactly, perfect!